api/src/main/java/edu/cornell/mannlib/vitro/webapp/rdfservice/impl/RDFServiceImpl.java

```diff
@@ -265,36 +278,54 @@ protected static String sparqlNode(Node node, String varName) {
         }
     }
 
-     // see http://www.python.org/doc/2.5.2/ref/strings.html
-     // or see jena's n3 grammar jena/src/org.apache/jena/n3/n3.g
-    protected static void pyString(StringBuffer sbuff, String s)  {
+    // see http://www.python.org/doc/2.5.2/ref/strings.html
+    // or see jena's n3 grammar jena/src/org.apache/jena/n3/n3.g
+    protected static void pyString(StringBuffer sbuff, String s) {
         for (int i = 0; i < s.length(); i++) {
             char c = s.charAt(i);
 
             // Escape escapes and quotes
-            if (c == '\\' || c == '"' )
-            {
-                sbuff.append('\\') ;
-                sbuff.append(c) ;
-                continue ;
+            if (c == '\\' || c == '"') {
+                sbuff.append('\\');
+                sbuff.append(c);
+                continue;
             }
 
             // Whitespace
-            if (c == '\n'){ sbuff.append("\\n");continue; }
-            if (c == '\t'){ sbuff.append("\\t");continue; }
-            if (c == '\r'){ sbuff.append("\\r");continue; }
-            if (c == '\f'){ sbuff.append("\\f");continue; }
-            if (c == '\b'){ sbuff.append("\\b");continue; }
-            if( c == 7 )  { sbuff.append("\\a");continue; }
+            if (c == '\n') {
+                sbuff.append("\\n");
+                continue;
+            }
+            if (c == '\t') {
+                sbuff.append("\\t");
+                continue;
+            }
+            if (c == '\r') {
+                sbuff.append("\\r");
+                continue;
+            }
+            if (c == '\f') {
+                sbuff.append("\\f");
+                continue;
+            }
+            if (c == '\b') {
+                sbuff.append("\\b");
+                continue;
+            }
+            if (c == 7) {
+                sbuff.append("\\a");
+                continue;
+            }
```

Comment:
I think its better if we implement a replacement map as:
Map<Character, String> replacements = new HashMap<>();
        replacements.put('\n', "\\n");
        replacements.put('\t', "\\t");
        replacements.put('\r', "\\r");
        replacements.put('\f', "\\f");
        replacements.put('\b', "\\b");
        replacements.put((char)7, "\\a");
        
and then just call sbuff.append(replacements.get(c));
saves us from using continue this many times, and is easily maintainable.